babel.config.js

```diff
@@ -71,7 +71,8 @@ module.exports = {
           !!(
             fileName.includes('-test') ||
             fileName.includes('-Test') ||
-            /\/node_modules\//.test(fileName)
+            /\/node_modules\//.test(fileName) ||
+            fileName.includes('.d.ts')
```

Comment:
We don't want to transpile the types files to TS as otherwise that defeats the point of having them